src/nvim/ex_cmds.c

```diff
@@ -6035,10 +6034,7 @@ FUNC_ATTR_NONNULL_ARG(1, 2, 3)
     }
 
     // computing the length of the column that will display line number
-    //TODO: Is log10 efficient for counting the nr of digits?
-    int col_width = log10(highest_num_line) + 1 + 3;
-    // better? log10 might not be precise for large numbers (ref?)
-    //int col_width = snprintf(NULL, 0, "%ld", highest_num_line) + 3;
+    int col_width = snprintf(NULL, 0, "%ld", highest_num_line) + 3;
```

Comment:
And here I was thinking it was overkill to include `math.h` and use this complicated function on floats.

I'm not sold on this in any way, I've just seen a comment somewhere on SO while investigating, that claimed that `log10` might not give proper return values for large integers. Can't find it anymore and haven't seen it mentioned anywhere else, so I might misremember that...